custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -259,7 +260,10 @@ class NikshayFollowupPayloadGenerator(BaseNikshayPayloadGenerator):
 
     def get_payload(self, repeat_record, test_case):
         occurence_case = get_occurrence_case_from_test(test_case.domain, test_case.get_id)
-        episode_case = get_open_episode_case_from_occurrence(test_case.domain, occurence_case.get_id)
+        episode_case = get_associated_episode_case_for_test(test_case.domain, test_case)
+        if not episode_case:
+            episode_case = get_episode_case_from_occurrence(test_case.domain, occurence_case.get_id,
```

Comment:
:+1: 
Do you think it would be useful to bundle this into the `get_associated_episode_case_for_test` method (or something similar in `case_utils`)? This seems like something we'd always want to use with the new case structure.